tests/hmm/hmumu_utils.py

```diff
@@ -649,7 +649,10 @@ def default_weight(n):
                         weights_selected,
                         use_cuda
                     )
-
+                    
+                    print("plot CIT dnn histos")
```

Comment:
Hi @nanlu06 , I was wondering if these print statements for debugging and can be removed now?

Reply:
done